pkg/lib/bundle/generate_test.go

```diff
@@ -303,3 +304,43 @@ func TestGenerateFunc(t *testing.T) {
 	require.NoError(t, err)
 	require.EqualValues(t, output, string(annotationsBlob))
 }
+
+func TestGenerateCSVlessBundle(t *testing.T) {
+	csvlessBundleDir := "testdata/csvless_bundle/prometheus_v0.22.2"
+
+	err := GenerateFunc(BundleDir(csvlessBundleDir), WithPackageName("prometheous"), WithChannels("alpha,beta"),
+		WithDefaultChannel("alpha"))
+	assert.NoError(t, err)
```

Comment:
`require` will fail the test immediately

Reply:
yes, but I want to keep the test going if this is wrong. I use Assert for test and require for well, require for need to fail now.

Reply:
I know the rest of the test suit never used assert, but I want this test to excersise all tests at once not to spit out error one at a time. Please let me know if you agree. Thanks.